packages/office-ui-fabric-react/src/components/Layer/Layer.styles.ts

```diff
@@ -0,0 +1,38 @@
+import { ILayerStyleProps, ILayerStyles } from './Layer.types';
+import {
+  IStyle,
+  ITheme,
+} from '../../Styling';
+
+export const getStyles = (
+  props: ILayerStyleProps
+): ILayerStyles => {
+  const {
+    className,
+    theme,
+    isNotHost
+  } = props;
+
+  // const { palette, semanticColors } = theme;
+
+  return ({
+    root: [
+      'ms-Layer',
+      isNotHost && 'ms-Layer--fixed' && {
+        position: 'fixed',
+        zIndex: 1000000,
+        top: 0,
+        left: 0,
```

Comment:
Left get changed to right automatically by getClassNames when the page is RTL. I don't believe this should impact anything but please just verify that it works in RTL.

Reply:
This looks fine in RTL mainly because the width is '100vw', so left:0 and right:0 are the same either way. The text automatically aligns to the right tho, but that's not part of this style set.